views/js/runner/plugins/navigation/review/review.js

```diff
@@ -268,10 +282,13 @@ define([
             //disabled by default
             this.disable();
 
+
             if (!isEnabled()) {
                 this.hide();
             }
 
+            togglePanel(testConfig.review.defaultOpen);
+
```

Comment:
To enforce my previous comment, here you force the display of the component, even if it has been previously disabled...